src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
```

Comment:
It's recommended to add error handling for DOM queries. If the `table` element is not found in the DOM, the script will throw an error at the line where you try to access its `rows` property. Consider checking if the `table` exists before proceeding with the rest of the script.